doc/rippled-example.cfg

```diff
@@ -659,14 +659,16 @@
 #       ...
 #
 #   Examples:
-#       type=RocksDB
-#       path=db/rocksdb
+#       type=HyperLevelDB
+#       path=db/hyperldb
```

Comment:
HyperLevelDB is listed as Deprecated, yet we use it in the example. How about:
type=RocksDB # NuDB is preferred on Windows